lib/ListTemplate/listTemplate.js

```diff
@@ -27,6 +30,9 @@ import { formatUserName } from '../utils/formatUserName';
  * }}
  * Note: check which params are required based on used columns
  */
+
+const { formatNumber } = useIntl();
```

Comment:
React hooks must be used inside React functions or custom hooks (as noticed by Sonar). It requires refactoring of listTemplate, or passing intl prop to it

Reply:
You can try `createIntl` for non-component
https://formatjs.io/docs/react-intl/api/#createintl